plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/content/ContentTypeResourceController.java

```diff
@@ -76,10 +92,10 @@ private void validateCreate(ContentTypeDtoRequest contentType, BindingResult bin
 
     @Override
     @RestAccessControl(permission = Permission.SUPERUSER)
-    public ResponseEntity<Void> delete(@PathVariable("code") String code) {
+    public ResponseEntity<SimpleRestResponse<Map>> delete(@PathVariable("code") String code) {
         logger.debug("REST request - delete content type {}", code);
         service.delete(code);
-        return ResponseEntity.ok().build();
+        return ResponseEntity.ok(new SimpleRestResponse<>(new HashMap<>()));
```

Comment:
This fix is incomplete. The api must return the code of the deleted content type. In particular the response body have to be something like this:
`
{
    "payload": {
        "code": "<contentTypeCode>"
    },
    "errors": [],
    "metaData": {}
 }
`